packages/wrangler/src/pages.tsx

```diff
@@ -783,7 +783,10 @@ export const pages: BuilderCallback<unknown, unknown> = (yargs) => {
 
         let miniflareArgs: MiniflareOptions = {};
 
-        let scriptReady = true;
```

Comment:
Yup, that's my bad. Must have changed it to `true` when testing and forgot to flip back.

And your Promise approach is much neater anyway.